src/core/defaults.ts

```diff
@@ -49,8 +49,8 @@ const DEFAULTS: Partial<Configuration> = {
   },
   recommendations: {
     location: {
-      distance: '10km',
-      minSize: 10,
+      distance: undefined,
```

Comment:
I think what you meant to do is set `location: false` rather than set the distance and minSize as undefined. I think if you set these to undefined, it will still try to use the user's location, just w/ these parameters set to undefined. Whereas I think what you're trying to do is make it not request user location by default. Please correct me if I'm wrong.

Reply:
So I had tried that but the types where never set to allow a Boolean there. I tried to hunt down where those declarations where but decided to do this for an interim fix.